components/generic_modal.scss

```diff
@@ -0,0 +1,122 @@
+@charset 'UTF-8';
+
+.modal {
+    .GenericModal {
+        &.modal-dialog {
+            margin-top: calc(50vh - 240px);
+        }
+
+        .modal-body {
+            max-height: 100%;
+            padding: 0;
+
+            .form-control {
+                border: 2px solid var(--button-bg);
+                box-sizing: border-box;
+                border-radius: 4px;
+                height: 40px;
+            }
+
+            .input-clear {
+                font-style: normal;
+                font-weight: normal;
+                font-size: 18px;
+                line-height: 16px;
+                color: var(--center-channel-color);
+                opacity: 0.48;
+                right: 36px;
+                top: 11px;
+                width: 16px;
+                height: 16px;
+            }
+        }
+
+        .modal-content {
+            border-radius: 8px;
+        }
+
+        .modal-header {
+            background: transparent;
+            border: none;
+            min-height: 0;
+            padding: 0;
+
+            .close {
+                color: var(--center-channel-color-56);
```

Comment:
Would you mind switching these transparent ones to the `-rgb` variants while you're in here?